index.bs

```diff
@@ -190,6 +190,10 @@ dictionary MediaStreamTrackProcessorInit {
 };
 </pre>
 
+Note: There is WG consensus that the interface should be exposed on DedicatedWorker.
+There is no WG consensus that the interface should not be exposed on Window.
```

Comment:
I'm a bit concerned this double-negative is hard to parse. I would prefer something like. We normally don't require consensus to *not* do something, so it would seem more correct o say "There is no WG consensus that the interface should be exposed on Window.".

Or doing the same as for audio would be: "There is no WG consensus on whether or not the interface should be exposed on Window."